### ISSUE TITLE
Hi, does this project support Chinese input and output? 

### ISSUE THREAD

**Issue by zhaodongyang2016** [marty1885/paroli#7]
I have a 3588 with Ubuntu installed. If the project supports Chinese, I'd like to give it a try. Thank you!

**Comment by marty1885** [comment]
No, I don't have a trained model for Chinese. (Plus the eSpeak phenomizer technically supports Chinese. It's kinda bad at it.)
